source/UIAHandler/__init__.py

```diff
@@ -1237,7 +1237,7 @@ def getNearestWindowHandle(self, UIAElement):
 		try:
 			window = new.cachedNativeWindowHandle
 		except COMError:
-			window = None
+			return None

```

Comment:
Note, this fixes two issues:

1. NVDA trying to call `getWindowHandleDebugString` on a None window handle. This was discovered while debugging the issue with Edge.
2. Saving a None _nearestWindowHandle on an UIA element

Reply:
can we a log something at debugWarning here?